src/MoBi.Presentation/UICommand/AddNewIndividualCommand.cs

```diff
@@ -0,0 +1,33 @@
+﻿using MoBi.Core.Domain.Model;
+using MoBi.Core.Services;
+using MoBi.Presentation.Tasks.Interaction;
+using OSPSuite.Core.Domain.Builder;
+using OSPSuite.Presentation.UICommands;
+
+namespace MoBi.Presentation.UICommand
+{
+   public class AddNewIndividualCommand : ObjectUICommand<IndividualBuildingBlock>
+   {
+      protected IPKSimStarter _pkSimStarter;
+      private readonly IMoBiContext _context;
+      private readonly IInteractionTasksForIndividualBuildingBlock _interactionTask;
+
+      public AddNewIndividualCommand(IPKSimStarter pkSimStarter, IMoBiContext context,
+         IInteractionTasksForIndividualBuildingBlock interactionTask)
+      {
+         _pkSimStarter = pkSimStarter;
+         _context = context;
+         _interactionTask = interactionTask;
+      }
+
+      protected override void PerformExecute()
+      {
+         var individualFromPKSim = _pkSimStarter.CreateIndividual();
```

Comment:
this can probably also be generic completely and use the type. 
pksimStarter.Create<Individual> etc.. not a lof of code so that's ok but this will be duplicated EXACTLY For each type we add. Not sure if we plan to load more from PKSim ( I could see molecule being one of them, events etc.. So making it reusable would make sense IMO